src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -311,7 +311,7 @@ def get_status_model(self) -> WorkflowRunModel:
             raise WorkflowRunNotStarted(message) from e
         return self._runtime.get_workflow_run_status(run_id)
 
-    def get_results(self, wait: bool = False) -> t.Sequence[t.Any]:
+    def get_results(self, wait: bool = False, force_as_sequence: bool = False) -> t.Any:
```

Comment:
IMO the multi-output behavior was already difficult to reason about. This option additionally increases the number of cases to consider by 2. Is there a good use case for it?

Reply:
My reasoning for it went like this: The CLI needs to know how many results you got back in order to format the output properly. At present it uses the len of the return from get_results to determine this. With these changes, if I have 1 result which is an iterable of len n, the cli would incorrectly report this as n results. Therefore we need a way for the CLI to be able to reliably determine the number of results. 

We could do this via private methods, however I could see a use-case where a user has written general-purpose code to handle the results of workflows, and would therefore run into the same ambiguity problem as the CLI. Therefore it made sense to me to have this be a public option. WDYT?

Reply:
Right, CLI needs to know if it deals with one or multiple values because it affects the formatting. I also agree that CLI should only use public APIs.

I like @jamesclark-Zapata's point that our goal should be to do whatever regular Python code would.

Perhaps we can the problem around? How would need the CLI work if we wanted to implement `orq wf results` for regular Python functions instead of workflows?

Reply:
Could it be as simple as:
```
results = run.get_results()
n_outputs = len(results) if isinstance(results, tuple) else 1
...
```

Reply:
I like the `if isinstance(x, tuple)`. I would worry about edge cases if we put this under the API implementation but if it's moved to the formatting code at the last layer (CLI) we should be fine.

Reply:
Okay I like this. As of [f102239](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/141/commits/f102239cf0349008eb95efd68ba26f419cc47589) `force_as_sequence` no longer exists. Instead the API behaves in line with vanilla python, and the CLI converts non-tuple values into 1-element tuples that can then be iterated over just like before.